actors/builtin/miner/miner_actor.go

```diff
@@ -780,8 +774,8 @@ func (a Actor) ConfirmSectorProofsValid(rt Runtime, params *builtin.ConfirmSecto
 				pwrTotal.QualityAdjPowerSmoothed, circulatingSupply)
 
 			totalPrecommitDeposit = big.Add(totalPrecommitDeposit, precommit.PreCommitDeposit)
+			initialPledge, replacedAge, replacedDayReward := replacedSectorParameters(rt, precommit, initialPledge, replacedBySectorNumber)
```

Comment:
This overwrite of `initialPledge` is subtle, and a bit confusing with the name `replacedSectorParameters`. It would be clearer if this just returned the replaced sectors pledge as `replacedPledge`, and the `max()` performed here.